styles/base.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
   Exclude:
     - spec/manageiq/**/*
     - vendor/**/*
+  SuggestExtensions: false
```

Comment:
Wow, is this the only change that we need to make to the config? i.e. no namespace changes or anything?

Reply:
I haven't gone through every repo in the cross repo test to ensure that nothing else in their local rubopcop config break, but yes with this change I could use the latest on the core repo

Reply:
I don't see any major problems in the rubocop changelog.  Let's give it a shot